ansible/roles/topology/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+# Nodes to be included in topology tree, must include all Slurm compute nodes
+topology_topology_nodes: []
+
+# If set, will override topology.conf file auto-detected from OpenStack project
+# topology_topology_override:
```

Comment:
Nah looking at the templating this is the wrong approach. Instead of having a massive if/else in the actual template just let users voerride the template entirely, i.e. replace this var with `topology_conf_template: topology.conf.j2`.

(I think actually in general we should ALWAYS allow users to override templates, gets you out of a lot of places you otherwise need to fork roles etc.)